helm/kvm-operator/ci/default-values.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Why is this needed?

Reply:
The file is needed to keep the new helm chart linting happy

The line might not be necessary, I just based this off of what's in use elsewhere